### DISCUSSION TITLE
Get  E576: Error while reading ShaDa file

### DISCUSSION THREAD

**Discussion by OriLiMu** [LazyVim/LazyVim#1853] [Q&A] [answered]
Here is my error msg when I enter nvim : 

```
E576: Error while reading ShaDa file: there is an item at position 5991 that must not be the
re: Missing items are for internal uses only
E576: Error while reading ShaDa file: there is an item at position 5991 that must not be the
re: Missing items are for internal uses only
```

I already check this [link](https://github.com/neovim/neovim/issues/6875). Which says remove the folder shada will be fine. 

But I cant find shada files in my nvim config. 

I try `fd shada` and find nothing both in ~/.local/share/nvim and ~/.config/nvim

How to fix it?

**Comment by dpetka2001** [comment, marked as answer]
`shada` directory is located at `~/.local/state/nvim/`.

**Reply by OriLiMu** [reply]
Thank you.

**Comment by Shavaizz** [comment]
I'm facing the same issue, instead I am on windows. Not sure how to deal with it. Kindly help me out.

**Reply by dpetka2001** [reply]
From inside Neovim, do `:echo stdpath("state")` and this will print out the `state` directory on every OS. Inside that directory is the `shada` directory. Don't use Windows, so don't know the exact path on Windows.

**Reply by Shavaizz** [reply]
Thanks, this worked for me.

**Reply by folke** [reply]
In my tests yesterday on windows, the problem is back after restarting windows.
It seems shada is just broken on windows right now.
Someone should report an issue in the NEovim repo (if it doesn't exist yet)
